modules/javafx.web/src/main/native/Source/cmake/OptionsMSVC.cmake

```diff
@@ -110,11 +110,19 @@ add_compile_options(
 )
 
 if (NOT WTF_CPU_X86)
+    if (PORT STREQUAL "Java")
+    # Create pdb files for debugging purposes, also for Release builds
```

Comment:
Now that this is in an "if...Java" block, the comment is misleading. Maybe change it to something like "Suppress creation of pdb files for Release builds" and also add a "FIXME" or "TODO" to re-enable them for debug builds?

Reply:
Done